From: Michael Hanke <mih@debian.org>
Subject: Disable gt*gahp

Debian maintainer does not yet know how to rebuild the necessary
.jar files.
--- a/src/gt42_gahp/CMakeLists.txt
+++ b/src/gt42_gahp/CMakeLists.txt
@@ -18,8 +18,9 @@
 
 if (HAVE_EXT_GLOBUS)
 
-	 install(FILES gt42-gahp.jar
-			 DESTINATION ${C_LIB} )
+	# dunno how to build this from the sources
+	#install(FILES gt42-gahp.jar
+	#		 DESTINATION ${C_LIB} )
 
 else()
 
--- a/src/gt4_gahp/CMakeLists.txt
+++ b/src/gt4_gahp/CMakeLists.txt
@@ -18,8 +18,9 @@
 
 if (HAVE_EXT_GLOBUS)
 
-	 install(FILES gt4-gahp.jar
-			 DESTINATION ${C_LIB} )
+	# dunno how to build this from source
+	# install(FILES gt4-gahp.jar
+	#		 DESTINATION ${C_LIB} )
 
 else()
 
